From bd6af975192afcfcf82c87bbdd146dc342430421 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 21 Sep 2022 17:15:11 +0200
Subject: [PATCH 219/841] feat(fdts): add hconf1 OTP reference for stm32mp251

Add the HCONF1 OTP register for stm32mp251.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e24fa124c17bf4a2d51ae4f35d150fc16f831ae8 feat(fdts): add hconf1 OTP reference for stm32mp251]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I92bd06fac7b20e25d3c586e096461bb8c9d0628d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/267511
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp251.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 5a890e86c..bb5ba111a 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -300,6 +300,9 @@
 			package_otp@1e8 {
 				reg = <0x1e8 0x1>;
 			};
+			hconf1_otp: otp124@1f0 {
+				reg = <0x1f0 0x4>;
+			};
 		};
 
 		iwdg1: watchdog@44010000 {
-- 
2.47.2

